katello.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "foreman-tasks", "~> 0.13", ">= 0.14.1"
   gem.add_dependency "dynflow", ">= 1.2.0"
   gem.add_dependency "foreman_docker", ">= 0.2.0"
+  gem.add_dependency "activerecord-import"
```

Comment:
this will require packaging :)

Reply:
Done! https://github.com/theforeman/foreman-packaging/pull/3432  :)

we'll still need a 3rd pr to update katello, but we can do that after both this and ^ are merged